tools/rpmkeys.c

```diff
@@ -75,7 +74,29 @@ int main(int argc, char *argv[])
 	break;
     /* XXX TODO: actually implement these... */
```

Comment:
Maybe delete this comment while at it? :smile: A fine example of why comments are so bad - they don't get updated along with the code.